src/Polly.Core/Telemetry/ResilienceStrategyTelemetry.cs

```diff
@@ -21,6 +21,22 @@ internal ResilienceStrategyTelemetry(ResilienceTelemetrySource source, Telemetry
 
     internal ResilienceTelemetrySource TelemetrySource { get; }
 
+ #pragma warning disable S4049
+ #pragma warning disable CA1024
+    /// <summary>
+    /// Returns the concatenated telemetry source.
+    /// </summary>
+    /// <returns>The concatenated telemetry source.</returns>
```

Comment:
I would say "concatenated" is an implementation detail - instead I'd say something like "Returns a string representation of the source of the telemetry".